src/custom/pages/Profile/index.tsx

```diff
@@ -223,13 +228,22 @@ export default function Profile() {
             </span>
           </BalanceDisplay>
           <CardActions>
-            <ExtLink href={getBlockExplorerUrl(chainId || 1, COW_CONTRACT_ADDRESS[chainId || 1], 'address')}>
+            <ExtLink
+              title="View contract"
+              href={getBlockExplorerUrl(chainId, COW_CONTRACT_ADDRESS[chainId], 'address')}
+            >
               Contract ↗
             </ExtLink>
 
-            <AddToMetamask currency={COW[chainId || 1] as Currency | undefined} />
+            {currencyCOW && library?.provider?.isMetaMask && <AddToMetamask shortLabel={true} currency={currencyCOW} />}
```

Comment:
boolean props that are always true can just be passed as such:
```suggestion
            {currencyCOW && library?.provider?.isMetaMask && <AddToMetamask shortLabel currency={currencyCOW} />}
```

Reply:
Ah that's right, thank you.